webhook/processConfig.ts

```diff
@@ -0,0 +1,239 @@
+/* eslint-disable complexity */
+import { setTimeout as setPromisedTimeout } from 'timers/promises'
+
+import { createFile as touch, readJSON, remove, writeJSON } from 'fs-promise'
+import execa, { ExecaChildProcess, ExecaSyncReturnValue } from 'execa'
+import CircularBuffer from 'circular-buffer'
+import Bugsnag from '@bugsnag/js'
+
+import {
+  downloadAndProcessGtfsFeed,
+  exists,
+  uploadAndDeleteLog,
+  wait,
+  writeStatus,
+  WebhookConfig,
+  Status,
+  PeliasFeed
+} from './utils'
+
+const {
+  BUGSNAG_APP_TYPE,
+  BUGSNAG_RELEASE_STAGE,
+  PELIAS_CONFIG_DIR,
+  PELIAS_UPDATE_LOCK_FILE,
+  WORKER_BUGSNAG_NOTIFIER_KEY
+} = require('../consts.json')
+
+const PELIAS_CONFIG_FILE = `${PELIAS_CONFIG_DIR}/pelias.json`
+
+Bugsnag.start({
+  apiKey: WORKER_BUGSNAG_NOTIFIER_KEY,
+  appType: BUGSNAG_APP_TYPE,
+  appVersion: require('../package.json').version,
+  releaseStage: BUGSNAG_RELEASE_STAGE
+})
+
+const status: Status = {
+  completed: false,
+  error: false,
+  message: 'Initializing...',
+  percentComplete: 0
+}
+
+;(async () => {
```

Comment:
It's not immediately obvious to me what this syntax implies. Could you add a comment here?

Reply:
Have added clarification in bd0dd663a5578f8c305459c775f3a7881844d257